### PR TITLE
Version to 1.0.0

### PR DESCRIPTION
**Pull request by jnturton**
Prepare master for a 1.0.0 tag and deployment of an 1.0.0 package to PyPI.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [egg_info]
-tag_build = dev
+tag_build =
```
Comment (cgivre) [inline setup.cfg:2]: Does this need to be here at all?  Should we replace with `production` or something like that?

File: setup.py [hunk 2]
```diff
@@ -25,20 +29,17 @@
     long_description = f.read()
 
 setup(name='sqlalchemy_drill',
-      version='0.3.0',
+      version='0.3.1',
```
Comment (cgivre) [inline setup.py]: What would you think about calling this version 1.0?

**Review by cgivre — COMMENTED** [review]
Two minor questions, but LGTM.

File: setup.py (hunk 2 above)
Comment (jnturton) [reply setup.py]: Yes good call, too many projects sit in denial in 0.x for years.  I will push this in a couple of minutes.

File: setup.cfg (hunk 1 above)
Comment (jnturton) [reply setup.cfg:2]: Whatever we put there will be glued onto the end of our package version number so to do a production release we want nothing.  Dropping that line entirely would probably also produce the same result by default, I just imitated what I found in some instructions online.  Seems explicit and harmless, if redundant.  Can remove the line if you prefer.

**Review by cgivre — APPROVED** [review]
LGTM +1

**Pull request merged**